lib/inventory_refresh/inventory_collection.rb

```diff
@@ -92,14 +92,8 @@ class InventoryCollection
     #   parameters
     attr_accessor :parent_inventory_collections
 
-    attr_reader :model_class, :strategy, :attributes_blacklist, :attributes_whitelist, :custom_save_block, :parent,
-                :internal_attributes, :delete_method, :dependency_attributes, :manager_ref, :create_only,
-                :association, :complete, :update_only, :transitive_dependency_attributes, :check_changed, :arel,
-                :inventory_object_attributes, :name, :saver_strategy, :targeted_scope, :default_values,
-                :targeted_arel, :targeted, :manager_ref_allowed_nil, :use_ar_object,
-                :created_records, :updated_records, :deleted_records, :retention_strategy,
-                :custom_reconnect_block, :batch_extra_attributes, :references_storage, :unconnected_edges,
-                :assert_graph_integrity
+    attr_accessor :attributes_blacklist, :attributes_whitelist
+    attr_reader :model_class, :strategy, :custom_save_block, :parent, :internal_attributes, :delete_method, :dependency_attributes, :manager_ref, :create_only, :association, :complete, :update_only, :transitive_dependency_attributes, :check_changed, :arel, :inventory_object_attributes, :name, :saver_strategy, :targeted_scope, :default_values, :targeted_arel, :targeted, :manager_ref_allowed_nil, :use_ar_object, :created_records, :updated_records, :deleted_records, :retention_strategy, :custom_reconnect_block, :batch_extra_attributes, :references_storage, :unconnected_edges, :assert_graph_integrity
```

Comment:
ETOOMANYATTRIBUTES :laughing:

Reply:
amusing that they felt scrolling off the side of the screen was better than making it so you can read it.

Almost feels like a passive aggressive way of them asking you to reduce the number of attributes.